src/agent/main.cpp

```diff
@@ -131,6 +117,22 @@ static otbrLogLevel GetDefaultLogLevel(void)
     return level;
 }
 
+static void PrintRadioVersion(otInstance *aInstance)
+{
+    printf("%s\n", otPlatRadioGetVersionString(aInstance));
+}
+
+static void PrintRadioVersionAndExit(const std::vector<const char *> &aRadioUrls)
+{
+    otbr::Ncp::ControllerOpenThread ncpOpenThread{/* aInterfaceName */ "", aRadioUrls, /* aBackboneInterfaceName */ "",
+                                                  /* aDryRun */ true};
+
+    SuccessOrDie(ncpOpenThread.Init(), "failed to initialize NCP");
```

Comment:
try to keep every log message unique? How about moving this log message into Init() method itself.

Reply:
that way, we can make `Init` return void.

Reply:
SGTM.

Reply:
done